spec/spec_helper.rb

```diff
@@ -15,10 +15,14 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 RSpec.configure do |config|
+  require 'webmock/rspec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.